### PR TITLE
Translate pgan document

### PR DESCRIPTION
**Pull request by nninept**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #4 

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
PGAN 모델의 문서를 새로 번역했습니다

### CODE REVIEW

File: facebookresearch_pytorch-gan-zoo_pgan.md [hunk 1]
```diff
@@ -22,49 +22,50 @@ demo-model-link: https://colab.research.google.com/drive/19NTYFNUT9js78UZ0g_3Isn
...
 
-In computer vision, generative models are networks trained to create images from a given input. In our case, we consider a specific kind of generative networks: GANs (Generative Adversarial Networks) which learn to map a random vector with a realistic image generation.
+컴퓨터 비전(computer vision)에서 생성자(generative) 모델이란, 주어진 입력값을 토대로 새로운 이미지를 만들어내는 신경망을 뜻합니다.
+지금 우리가 다루는 것은 생성자 신경망의 특정 모델로서: 무작위의 벡터를 사실적인 이미지로 변환시켜주는 GANs (Generative Adversarial Networks) 모델 입니다.
 
-Progressive Growing of GANs is a method developed by Karras et. al. [1] in 2017 allowing generation of high resolution images. To do so, the generative network is trained slice by slice. At first the model is trained to build very low resolution images, once it converges, new layers are added and the output resolution doubles. The process continues until the desired resolution is reached.
+GAN의 점진적 학습(Progressive Growing of GANs)은 Karras와 그 외[1]가 2017년 발표한 방법론으로, 고해상도의 이미지를 생성할 수 있습니다. 이를 위해 생성자 모델은 부분적으로 차근차근 훈련을 하게됩니다. 처음에는 가장 낮은 해상도를 학습하도록 하고, 어느정도 모델이 수렴하게 되면, 새로운 계층(layer)이 모델에 더해져, 기존 출력 이미지 크기의 2배로 커진 출력값으로 이어서 훈련하는 방식입니다. 이 과정은 원하는 해상도에 도달할때까지 반복됩니다.
 
-### Requirements
+### 준비물
```
Comment (cpprhtn) [inline facebookresearch_pytorch-gan-zoo_pgan.md:65]: `요구 사항` 이 더 어울리는거 같습니다!!

File: facebookresearch_pytorch-gan-zoo_pgan.md [hunk 2]
```diff
@@ -22,49 +22,50 @@ demo-model-link: https://colab.research.google.com/drive/19NTYFNUT9js78UZ0g_3Isn
 import torch
 use_gpu = True if torch.cuda.is_available() else False
 
-# trained on high-quality celebrity faces "celebA" dataset
-# this model outputs 512 x 512 pixel images
+# 유명인들의 고해상도 얼굴사진들로 만든 "celebA" 데이터셋으로 훈련시켰습니ㅏ
```
Comment (cpprhtn) [inline facebookresearch_pytorch-gan-zoo_pgan.md:25]: 오타 발견했습니다!

Comment (cpprhtn) [reply facebookresearch_pytorch-gan-zoo_pgan.md:25]: 글 전반적으로 온점(마침표)가 있다가 없다가 합니다.
통일해주면 더 깔끔할 것 같습니다!

**Review by cpprhtn — COMMENTED** [review]
오타가 좀 있군요! 수정해주시면 감사하겠습니다 :D

File: facebookresearch_pytorch-gan-zoo_pgan.md (hunk 2 above)
Comment (hyoyoung) [inline facebookresearch_pytorch-gan-zoo_pgan.md:25]: 마지막 글자 오타 수정 부탁드립니다

File: facebookresearch_pytorch-gan-zoo_pgan.md [hunk 3]
```diff
@@ -22,49 +22,50 @@ demo-model-link: https://colab.research.google.com/drive/19NTYFNUT9js78UZ0g_3Isn
...
+# 이 모델은 256x5126 픽셀 크기의 이미지를 출력하게 됩니다
 # model = torch.hub.load('facebookresearch/pytorch_GAN_zoo:hub',
 #                        'PGAN', model_name='celebAHQ-256',
 #                        pretrained=True, useGPU=use_gpu)
 ```
 
-The input to the model is a noise vector of shape `(N, 512)` where `N` is the number of images to be generated.
-It can be constructed using the function `.buildNoiseData`.
-The model has a `.test` function that takes in the noise vector and generates images.
+모델의 입력값으로 들어가는 노이즈(noise) 벡터는 `(N, 512)` 크기를 갖는데, 이때 `N` 은 생성하고싶은 데이터의 수입니다.
```
Comment (hyoyoung) [inline facebookresearch_pytorch-gan-zoo_pgan.md:36]: 이때 `N` 은 생성하고싶은 데이터의 수입니다.
-> 
이때 `N`개의 이미지가 생성 됩니다

File: facebookresearch_pytorch-gan-zoo_pgan.md [hunk 4]
```diff
@@ -22,49 +22,50 @@ demo-model-link: https://colab.research.google.com/drive/19NTYFNUT9js78UZ0g_3Isn
...
+# 생성한 이미지들을 torchvision과 matplotlib을 이용해서 시각화를 해봅시다
 import matplotlib.pyplot as plt
 import torchvision
 grid = torchvision.utils.make_grid(generated_images.clamp(min=-1, max=1), scale_each=True, normalize=True)
 plt.imshow(grid.permute(1, 2, 0).cpu().numpy())
 # plt.show()
 ```
 
-You should see an image similar to the one on the left.
+결과를 확인해보면 , 오른쪽의 이미지와 비슷한 사진을 확인 할 수 있을겁니다
```
Comment (hyoyoung) [inline facebookresearch_pytorch-gan-zoo_pgan.md:54]: 왼쪽과 비슷한 이미지를 볼수 있다는 뜻일까요?

**Review by hyoyoung — CHANGES_REQUESTED** [review]
전반적으로 잘되어있고 고생하셨습니다.
몇가지 사항 확인 부탁드립니다
